shortest_superseq_problem/shortest_superseq.cpp

```diff
@@ -0,0 +1,36 @@
+#include "shortest_superseq.h"
+
+SubArray ShortestSuperseq(const std::vector<int> &short_array,
+                          const std::vector<int> &long_array) {
+  absl::flat_hash_set<int> search_seq;  // search elements
+  size_t nxt_out = 0;                   // start of window
+  size_t nxt_in = 0;                    // next of end of window
+  size_t min_start = 0;
+  size_t min_end = long_array.size();
+
+  for (int i : short_array) search_seq.insert(i);
```

Comment:
you could make it a tiny bit more compact by initializing the hash map from the array: 
absl::flat_hash_set<int> search_seq(short_array.begin(), short_array.end());